src/app/components/team/TeamReport/TeamReportComponent.js

```diff
@@ -311,7 +311,7 @@ const TeamReportComponent = ({ team, setFlashMessage }) => {
                 InputProps={{
                   startAdornment: (
                     <InputAdornment position="start">
-                      <WhatsAppIcon style={{ color: 'var(--whatsappGreen)' }} />
+                      <WhatsAppIcon style={{ color: 'var(--whatsappGreen)', fontSize: '20px' }} />
```

Comment:
Just a question, is this inline style how we want to change font sizes on icons in the future or was this mostly a case of you sticking with the same basic code for now?

Reply:
yeah, this is just sticking to the basic code for now. I think the better pattern in the future is to continue down the stylesheet path that we started. Just wanted to keep this PR more self contained